deploy/config/deploy/yum.rb

```diff
@@ -63,4 +64,8 @@
       run "if [ -e #{repo_instance_path}-previous ]; then rm -rf #{repo_instance_path}-previous; fi" if overwrite || merge
     end
   end
+
+  task :purgecdn do
+    run "find #{repo_path} | sed 's|#{deploy_to}|https://yum.theforeman.org/|' | xargs curl -X PURGE -H 'Fastly-Soft-Purge:1'"
```

Comment:
it would be nice if we could only purge things that changed, not *everything*

Reply:
one possibility would be to use `find … -mtime -3` to find any files that have been modified in the last three days.
as the modification time is pulled from Koji (via rsync), it might be older than "just now".